lib/logstash/filters/translate.rb

```diff
@@ -128,23 +142,53 @@ def register
     end
   end # def register
 
-  public
-  def load_yaml(registering=false)
-    if !File.exists?(@dictionary_path)
-      @logger.warn("dictionary file read failure, continuing with old dictionary", :path => @dictionary_path)
+  private
+  def load_file(registering,fileName,yamlData=nil)
```

Comment:
in ruby, we don't use camelCase syntax, we favor underscore_case, see ```@dictionary_path``` for example. Can you please update your variables?